### ISSUE TITLE
Create tool for migrating legacy colors/typography to the new system

### ISSUE THREAD

**Issue by JeanPaiva** [tomusborne/generatepress#217]
### Description
With the introduction of the new colors/typography systems, we must update the site-library for the new settings format. Since the library is quite big will require a lot of manual work to migrate.

We can create an automatic tool to facilitate this process. The tool will be able to auto-create global colors, element colors, and default typographies.

@tomusborne what do you think? We can start working on this right after the release of 3.1.0.

**Comment by tomusborne** [comment]
This would be a massive time-saver. Although we might want it closer to the release of 3.1.0 (at least typography), or we'll be introducing new users who are importing sites to the old typography system vs the new one.

**Comment by JeanPaiva** [comment]
Yeah, you are right, we need this before the release for new users. We can work on this while on alpha/beta phases of the release.

**Comment by tomusborne** [comment]
Thinking about this. We could attach the typography migration to `active_callback` when users switch to the new system: https://github.com/tomusborne/generatepress/blob/release/3.1.0/inc/customizer.php#L1509

Means it wouldn't require anything special from the user - it would just do it for them. Of course, not everything can be migrated (like the ability to know if Google fonts are being used), so it would still require some set up from the user.

Not sure if it's the best idea - could definitely be better ones.

**Comment by JeanPaiva** [comment]
We can try this, I think I'm going to create a class for this migration so we can use it anywhere, even a standalone plugin if we decide to.
